netstandard/ref/System.Threading.cs

```diff
@@ -380,6 +383,7 @@ public sealed partial class ReaderWriterLock : System.Runtime.ConstrainedExecuti
         public void AcquireWriterLock(System.TimeSpan timeout) { }
         public bool AnyWritersSince(int seqNum) { throw null; }
         public void DowngradeFromWriterLock(ref System.Threading.LockCookie lockCookie) { }
+        ~ReaderWriterLock() { }
```

Comment:
This should not be included in NS when the type is sealed

Reply:
IIRC I did manually exclude this in NS2.0 and we should exclude them.

Reply:
Done